Gemfile

```diff
@@ -6,6 +6,7 @@ source "https://rubygems.org"
 
 gem "rails", "~> 5.2"
 
+gem "aws-sdk-s3"
```

Comment:
Do you want to set require: false on this as suggested in https://edgeguides.rubyonrails.org/active_storage_overview.html#amazon-s3-service ?

Reply:
Also good to pin it to version 1 - even though Rails is a bit less cautious

Reply:
I don't think it's necessary - it's always loaded anyway. Good point with the versioning - apparently it's not in my internal linter yet.